### PR TITLE
Army claims

### PR DESCRIPTION
**Pull request by aileendds**
## Checklist

I have…

- [x] run the application locally (`bin/rails server`) and verified that my changes behave as expected.
- [x] run static code analysis (`bin/rubocop`) and vulnerability scan (`bin/brakeman`) against my changes.
- [x] run the test suite (`bin/rake spec`) and verified that all tests pass.
- [x] summarized below my changes and noted which issues (if any) this pull request fixes or addresses.
- [x] thoroughly outlined below the steps necessary to test my changes.
- [x] attached screenshots illustrating relevant behavior before and after my changes.

## Summary of Changes

This pull request…
1. Adds some army conus claims text as requested by #188
2. Removes empty table rows used for adding margins.
3. Switches addresses to not be wrapped in <p> which adds extraneous line spacing and extra margins at the bottom
4. Removes id from service_specific_information_path() calls cause it's cleaner and I like it...

## Testing

To verify the changes proposed in this pull request…

To verify the changes proposed in this pull request…
clone this repo,
git checkout army_claims,
set up development dependencies according to CONTRIBUTING.md,
run bin/rails server, and
load up http://localhost:3000/customer-service

## Screenshots

<details><summary>Desktop</summary>

screencapture-localhost-3000-customer-service-1512503152790

</details>

<details><summary>Mobile</summary>

screencapture-localhost-3000-customer-service-1512503164765

</details>

### CODE REVIEW

File: app/views/customer_service/index.html.erb [hunk 1]
```diff
@@ -270,12 +270,14 @@
                     <% if contact.claims_post %>
                       <tr>
                         <th scope="row">Address:</th>
-                        <td><%= simple_format(contact.claims_post) %></td>
+                        <td><%= simple_format(contact.claims_post, {}, wrapper_tag: 'span') %></td>
```
Comment (jgarber623-gov) [inline app/views/customer_service/index.html.erb]: `simple_format` will add paragraph tags (a block-level element) around text and `wrapper_tag` is wrapping _those_ in a `<span>` (an inline element) which creates a functional-but-kind-of-weird markup structure:

```html
<span>
  <p>…</p>
</span>
```

Comment (aileendds) [reply app/views/customer_service/index.html.erb]: I checked the markup and it doesn't do that! Maybe an older version did?

Comment (aileendds) [reply app/views/customer_service/index.html.erb]: screen shot 2017-12-06 at 9 33 51 am

Comment (jgarber623-gov) [reply app/views/customer_service/index.html.erb]: Oh, derp. That's on me. I misinterpreted the documentation. _Carry on!_

**Pull request merged**